File: A2OJ/Ladder4/ladder4q41.py
def coprime(n1,n2):
    valid=1
    i=2
    while(i<=30000):
        if(n1%i==0 and n2%i==0):
            valid=0
            break
        i+=1
    if valid==1:
        return 1
    else:
        return 0
listing=[int(x) for x in raw_input().split()]
l=listing[0]
r=listing[1]
i=l
j=l+1
k=l+2
while i<=r-2:
    while j<=r-1:
        while k<=r:
            if(coprime(i,j)==1 and coprime(j,k)==1 and coprime(i,k)==0):
                print i,j,k
                exit()
            k+=1
        j+=1
    i+=1
print -1
exit();

File: A2OJ/Ladder4/ladder4q41.c
#include<stdio.h>
int coprime(long n1, long n2);
int main()
{
  long a,b,c,l,r,i,j,k;
  scanf("%ld %ld", &l, &r);
  for(i=l;i<=r-2;++i)
  {
    for(j=l+1;j<=r-1;++j)
    {
      for(k=l+2;k<=r;++k)
      {
        if(coprime(i,j)==1 && coprime(j,k)==1 && coprime(i,k)==0)
        {
          // printf("%d %d %d\n", coprime(i,j), coprime(j,k), coprime(i,k));
          printf("%ld %ld %ld", i, j, k);
          return 0;
        }
      }
    }
  }
  printf("-1");
  return 0;
}
int coprime(long n1, long n2)
{
  // int min=(n1>n2)?n2:n1;
  int valid=1;
  for(int i=2;i<=30;++i)
  {
    if(n1%i==0 && n2%i==0)
    {
      valid=0;
      break;
    }
  }
  if(valid==1)
  return 1;
  else
  return 0;
}
